SampleGallery/Samples/SDK 17134/RadialGradients/RadialGradients.xaml.cs

```diff
@@ -0,0 +1,310 @@
+                _scale = _compositor.CreateVector3KeyFrameAnimation();
+                _scale.InsertKeyFrame(0, new Vector3(0, 0, 0));
+                _scale.InsertKeyFrame(1, new Vector3(1, 1, 0));
+                _scale.Duration = TimeSpan.FromSeconds(1);
+                _scale.IterationCount = 50;
+
+                _pulseVisual.StartAnimation("Scale", _scale);
+
+                // Set up the stopwatch to reset each it is started again
+                _dt = new DispatcherTimer();
+                _stopwatch = new Stopwatch();
+                _dt.Interval = TimeSpan.FromMilliseconds(0.00f);
+                _dt.Tick += Dt_Tick;
+                _dt.Start();
+                _stopwatch.Start();
+
+                // Set boolean flag for if the stopwatch animation is on to true
+                animationOn = true;
+            }
+            else
+            {
+                // Button turns the colors of it's gradient stops back to cooler colors when it is shut off
+                _changeButtonGradientStop1 = _compositor.CreateColorKeyFrameAnimation();
+                _changeButtonGradientStop1.InsertKeyFrame(0, _indianRed);
+                _changeButtonGradientStop1.InsertKeyFrame(1, _blueViolet);
+                _changeButtonGradientStop1.Duration = TimeSpan.FromSeconds(2);
+
+                _BBGradientStop1.StartAnimation(nameof(_BBGradientStop1.Color), _changeButtonGradientStop1);
+
+                _changeButtonGradientStop2 = _compositor.CreateColorKeyFrameAnimation();
+                _changeButtonGradientStop2.InsertKeyFrame(0, _orangeRed);
+                _changeButtonGradientStop2.InsertKeyFrame(1, _lightBlue);
+                _changeButtonGradientStop2.Duration = TimeSpan.FromSeconds(2);
+
+                _BBGradientStop2.StartAnimation(nameof(_BBGradientStop2.Color), _changeButtonGradientStop2);
+
+                // Animations for the pulsing effect are turned off
+                _stop1OffsetAnim = _compositor.CreateScalarKeyFrameAnimation();
+                _stop1OffsetAnim.InsertKeyFrame(0, 0);
+                _stop1OffsetAnim.InsertKeyFrame(0f, 0f);
+                _stop1OffsetAnim.Duration = TimeSpan.FromSeconds(1.5f);
+                _stop1OffsetAnim.IterationCount = 50;
+                
+                _PBGradientStop1.StartAnimation("Offset", _stop1OffsetAnim);
+
+                _stop2offsetAnim2 = _compositor.CreateScalarKeyFrameAnimation();
+                _stop2offsetAnim2.InsertKeyFrame(0, 0);
+                _stop2offsetAnim2.InsertKeyFrame(0f, 0f);
+                _stop2offsetAnim2.Duration = TimeSpan.FromSeconds(1.5f);
+                _stop2offsetAnim2.IterationCount = 50;
+                _stop2offsetAnim2.DelayTime = TimeSpan.FromSeconds(.25f);
+
+                _PBGradientStop2.StartAnimation("Offset", _stop2offsetAnim2);
+
+                _pulseColor = _compositor.CreateColorKeyFrameAnimation();
+                _pulseColor.InsertKeyFrame(0, _transparent);
```

Comment:
What is the point of an animation that goes from transparent to transparent? That's just the same as no animation. Some comments on these would reduce confusion for the reader.